### DISCUSSION TITLE
identifying DEGs with integrated data

### DISCUSSION THREAD

**Discussion by GreyRockIQ** [satijalab/seurat#5488] [Q&A] [answered]
Hello
I have errors with identifying DEGs with integrated data as below with code

t.cells <- subset(immune.combined, idents = "CD4 Naive T")
Idents(t.cells) <- "stim"
avg.t.cells <- as.data.frame(log1p(AverageExpression(t.cells, verbose = FALSE)$RNA))
avg.t.cells$gene <- rownames(avg.t.cells)

Errors as Couldn't find functionAverageExpression

Any suggestions?
Thanks
GreyRock

**Comment by samuel-marsh** [comment, marked as answer]
Hi @GreyRockIQ 

Have you run `library(Seurat)` at the start of your R session?

Best,
Sam

**Reply by GreyRockIQ** [reply]
Hi @samuel-marsh 
Thanks
Worked with restart and reload
Thanks
GreyRock
